tasks1.js

```diff
@@ -0,0 +1,118 @@
+// 1
+function convertMinutesToHours(minutes) {
+  return (Math.floor(minutes / 60)) + ':' + (minutes % 60);
+}
+console.log(convertMinutesToHours(75));
+
+
+// 2
+function isTextUppercase(text) {
+  return text.toUpperCase() === text;
+}
+console.log(isTextUppercase('START'));
+
+
+// 3
+function isDivisible(number1, number2, number3) {
+  return number1 % number2 === 0 && number1 % number3 === 0;
+}
+console.log(isDivisible(6,3,3));
+
+
+// 4
+function getBillboardPrice(word, price) {
+  const wordLength = word.trim().length;
+  return wordLength * price;
+}
+
+console.log(getBillboardPrice('Hello world!', 10));
+
+
+// 5
+function makeOppositeNumer(number) {
+  return number - (number * 2);
+}
+console.log(makeOppositeNumer(45));
+
+
+// 6
+function makeNegativeWithCheck(num) {
+  if (num < 0){
+    return num
+  } else {
+    return -num
+  }
+}
+console.log(makeNegativeWithCheck(45));
+
+
+// 7
+function removeChar(str){
+  const strLength = (str.trim().length)-1;
+  return str.substring(1, strLength);
+}
+console.log(removeChar('START'));
+
+// 8
+function basicOp(operation, value1, value2){
+  if(operation === "+") {
+    return value1 + value2
+  }
+  if(operation === "-") {
+    return value1 - value2
+  }
+  if(operation === "*") {
+    return value1 * value2
+  }
+  if(operation === "/") {
+    return value1 / value2
+  }
+}
+console.log(basicOp("+", 4, 8));
+
+// 9
+function century(year) {
```

Comment:
You could solve it without an `if` statement by using `Math.ceil` instead of `Math.floor`